include/psp2/kernel/processmgr.h

```diff
@@ -96,6 +99,29 @@ SceUInt32 sceKernelGetProcessTimeLow(void);
  */
 SceUInt64 sceKernelGetProcessTimeWide(void);
 
+SceUID sceKernelGetCurrentProcess(void);
+SceInt32 sceKernelGetRemoteProcessTime(SceUID processId, SceKernelSysClock *pClock);
+int _sceKernelExitProcessForUser(SceInt32 stat);
+
+SceUID sceKernelGetStderr(void);
+SceUID sceKernelGetStdin(void);
+SceUID sceKernelGetStdout(void);
+
+const void * sceKernelGetProcessParam(void);
+
+SceKernelClock sceKernelLibcClock(void);
+SceKernelTime sceKernelLibcTime(SceKernelTime *pTime);
+
+typedef struct SceKernelTimeval {
+	SceUInt64 value;
+} SceKernelTimeval;
+
+typedef struct SceKernelTimezone {
+	SceUInt64 value;
+} SceKernelTimezone;
+
+int sceKernelLibcGettimeofday(SceKernelTimeval *p, SceKernelTimezone *z);
```

Comment:
```suggestion
int sceKernelLibcGettimeofday(SceKernelTimeval *tv, SceKernelTimezone *tz);
```
https://man7.org/linux/man-pages/man2/settimeofday.2.html

(or tp, tzp.
https://pubs.opengroup.org/onlinepubs/009604599/functions/gettimeofday.html)